cmake/std/atdm/utils/set_build_options.sh

```diff
@@ -227,6 +227,8 @@ fi
 # Use -fPIC or not
 if atdm_match_buildname_keyword fpic; then
   export ATDM_CONFIG_FPIC=ON
+elif atdm_match_buildname_keyword no-fpic; then
```

Comment:
This needs to be moved before the match for 'fpic' or it will not ever match 'no-fpic'.